<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getPassword</class>
 <widget class="QDialog" name="getPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>801</width>
     <height>247</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="psw_label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 20pt &quot;Segoe UI Variable Display Semib&quot;;</string>
      </property>
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="psw_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 5px solid rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 600 24pt &quot;Segoe UI Variable Display Semib&quot;;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="pswAgain_label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 20pt &quot;Segoe UI Variable Display Semib&quot;;</string>
      </property>
      <property name="text">
       <string>Повторите пароль:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="pswAgain_lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px solid rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 600 24pt &quot;Segoe UI Variable Display Semib&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="errorPsw_label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Build2/img/img.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="next_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(33, 32, 66);</string>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>851</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="regist_label">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 24pt &quot;Segoe UI Variable Display Semib&quot;;</string>
      </property>
      <property name="text">
       <string>Регистрация            </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="styleSheet">
         <string notr="true">font: 600 15pt &quot;Segoe UI Variable Display Semib&quot;;</string>
        </property>
        <property name="text">
         <string>имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userName_label">
        <property name="styleSheet">
         <string notr="true">font: 600 10pt &quot;Segoe UI Variable Display Semib&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="lookpsw_Button">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>130</y>
     <width>75</width>
     <height>65</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>65</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid  rgb(24, 205, 255);
border-radius: 30px;
background-color: transfert;
font: 600 10pt &quot;Segoe UI Variable Display Semib&quot;;</string>
   </property>
   <property name="text">
    <string>показать</string>
   </property>
  </widget>
  <zorder>formLayoutWidget</zorder>
  <zorder>next_Button</zorder>
  <zorder>back_Button</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>lookpsw_Button</zorder>
  <zorder>background_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
